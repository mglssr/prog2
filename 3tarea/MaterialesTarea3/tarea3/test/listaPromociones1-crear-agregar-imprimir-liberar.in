crearListaPromociones
imprimirListaPromociones
liberarListaPromociones
crearPromocion 02/01/2002 01/06/2002 10 1
agregarAPromocion 10 0 1 2 3 4 5 6 7 8 9
crearListaPromociones
agregarAListaPromociones
imprimirListaPromociones
crearPromocion 01/01/2002 01/06/2002 10 2
agregarAPromocion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaPromociones
imprimirListaPromociones
liberarListaPromociones
Fin
