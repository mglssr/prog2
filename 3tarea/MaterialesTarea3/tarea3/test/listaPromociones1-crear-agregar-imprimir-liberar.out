Bienvenido al programa principal de MercadoFinger. Por favor ingrese su comando:
1> Lista de promociones creada con exito.
2> 3> Lista promociones liberada con exito.
4> Promocion creada con exito.
5> Productos agregados a la promocion de forma exitosa.
6> Lista de promociones creada con exito.
7> Se ha agregado la promocion a la lista de forma exitosa.
8> Promocion #1 del 2/1/2002 al 1/6/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
9> Promocion creada con exito.
10> Productos agregados a la promocion de forma exitosa.
11> Se ha agregado la promocion a la lista de forma exitosa.
12> Promocion #2 del 1/1/2002 al 1/6/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
Promocion #1 del 2/1/2002 al 1/6/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
13> Lista promociones liberada con exito.
14> Fin.
